### PR TITLE
Config Sample update for openidconnect

### PR DESCRIPTION
**Pull request by mmattel**
## Description
Update config.apps.sample to reflect recent changes of openidconnect 2.2.0

## Related Issue
- References: https://github.com/owncloud/docs-server/issues/665

## Motivation and Context
Completeness of docs

## How Has This Been Tested?
Text change only

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [x] Documentation ticket raised: https://github.com/owncloud/docs-server/issues/665
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: config/config.apps.sample.php [hunk 1]
```diff
@@ -436,6 +443,13 @@
 	'insecure' => true
 ],
 
+/**
+ * Authenticate guests only with basic auth
```
Comment (EParzefall) [inline config/config.apps.sample.php:447]:
```suggestion
 * Authenticate guests only with basic auth.
```

File: config/config.apps.sample.php [hunk 2]
```diff
@@ -436,6 +443,13 @@
...
+ * Allows guests only to be able to log in using basic auth. Other users will need to use
```
Comment (EParzefall) [inline config/config.apps.sample.php:448]:
```suggestion
 * Guests are only able to log in using basic auth. Other users will need to use
```

**Review by EParzefall — APPROVED** [review]

File: config/config.apps.sample.php (hunk 1 above)
Comment (mmattel) [reply config/config.apps.sample.php:447]: Not committing this as this is rendered as headline and headlines are without trailing periods

Comment (jnweiger) [reply config/config.apps.sample.php:447]: It is not 'Authenticate guests only ...` -- it is `Authenticate only guests ...`

(but close enough :-) )

**Review by jnweiger — COMMENTED** [review]
approved!

File: config/config.apps.sample.php (hunk 1 above)
Comment (mmattel) [reply config/config.apps.sample.php:447]: This was discussed upfront with the female mastermind of English language and the decision was made as written 😅

**Pull request merged**